client/scss/media.scss

```diff
@@ -40,4 +40,13 @@
 	content-row main article .author-image {
 		display: none;
 	}
+
+	content-row main article aside {
```

Comment:
Any reason not to nest this with `content-row main article` and `content-row` above?